sql/src/main/java/io/crate/planner/operators/Order.java

```diff
@@ -99,6 +104,17 @@ public ExecutionPlan build(PlannerContext plannerContext,
         return executionPlan;
     }
 
+    private static void ensureOrderByColumnsArePresentInOutputs(List<Symbol> outputs, List<Symbol> orderBySymbols) {
+        Set<Symbol> columnsInOutputs = extractColumns(outputs);
+        for (Symbol columnInOrderBy : extractColumns(orderBySymbols)) {
+            if (!columnsInOutputs.contains(columnInOrderBy)) {
+                throw new UnsupportedOperationException(SymbolFormatter.format(
```

Comment:
just double checking - ordering by an ordinal will not fail this right?

Reply:
We resolve ordinals in the analyzer so at this point we've fields or references